src/scripts/configs/embeddables.coffee

```diff
@@ -0,0 +1,44 @@
+define (require) ->
+
+  settings = require('settings')
+
+  return {
+    embeddableTypes : [{
+      match : '#terp-'
+      matchType : 'a'
+      embeddableType : 'terp'
+      apiUrl : () ->
```

Comment:
There's a bunch of files in this commit with a space between the property and colon.  You may want to do a search replace for ` : ` -> `: `.